### PR TITLE
Change the maxage for a specific session

### PR DESCRIPTION
**Pull request by amay**
This change was originally discussed in https://github.com/expressjs/cookie-session/pull/22. It looks like that PR was closed without an objection to the change. Can this be merged, and if not, what can we do to get it merged?

### CODE REVIEW

**Comment by dougwilson** [comment]
Hi! Can you check on why the tests are failing, please :) ?

File: index.js [hunk 1]
```diff
@@ -209,6 +209,38 @@ Session.prototype.__defineGetter__('populated', function(){
 });
 
 /**
+ * Return the cookie options
+ *
+ * @return {Object}
+ * @api public
+ */
+
+Session.prototype.__defineGetter__('opts', function(){
```
Comment (dougwilson) [inline index.js]: Please update to not use the old defineGetter syntax

**Comment by amay** [comment]
Yep! Looks like the access pattern to sessionOptions changes (it used to be directly on the context and now you get it off the request).

**Comment by amay** [comment]
Okay. Tests and coverage are passing now.

**Comment by amay** [comment]
@dougwilson does  this look okay for merge now that everything is passing?

**Comment by amay** [comment]
@dougwilson any update on getting this merged? We would like to stop having to point to the github version. Thanks.

**Comment by dougwilson** [comment]
Hi @amay , sorry this PR got lost in my abyss :( I took a look, and I'm not certain if this even needs to be added, because you can just do `req.sessionOptions.maxAge = 3600`, as that is the purpose of the `req.sessionOptions` object. Mixing in a getter/setting in the session object directly is convenient, but just takes away more namespace from people's own code.

**Pull request closed without merging**